src/Features/Core/Portable/ReplaceMethodWithProperty/ReplaceMethodWithPropertyCodeRefactoringProvider.cs

```diff
@@ -95,14 +95,20 @@ private static bool HasGetPrefix(SyntaxToken identifier)
 
         private static bool HasGetPrefix(string text)
         {
-            return text.StartsWith(GetPrefix) && text.Length > GetPrefix.Length && !char.IsLower(text[GetPrefix.Length]);
+            return HasPrefix(text, GetPrefix);
+        }
+        private static bool HasPrefix(string text, string prefix)
+        {
+            return text.StartsWith(prefix, StringComparison.OrdinalIgnoreCase) && text.Length > prefix.Length && !char.IsLower(text[prefix.Length]);
```

Comment:
So, i'm a little unclear on when we should use each different comparer.  I'm pinging @davkean about this since he was dealing with this with the Turkish I problem.

Hey @davkean.  Could we write up a page somewhere that describes in what circumstances you'd want to use each specific type of comparer (including possibly 'never').